standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -878,12 +881,47 @@ public static ConfVars getMetaConf(String name) {
     THRIFT_FAILURE_RETRIES("metastore.failure.retries", "hive.metastore.failure.retries", 1,
         "Number of retries upon failure of Thrift metastore calls"),
     THRIFT_URIS("metastore.thrift.uris", "hive.metastore.uris", "",
-        "Thrift URI for the remote metastore. Used by metastore client to connect to remote metastore."),
+        "URIs Used by metastore client to connect to remotemetastore\n." +
+                "If dynamic service discovery mode is set, the URIs are used to connect to the" +
+                " corresponding service discovery servers e.g. a zookeeper. Otherwise they are " +
+                "used as URIs for remote metastore."),
+    THRIFT_SERVICE_DISCOVERY_MODE("metastore.service.discovery.mode",
+            "hive.metastore.service.discovery.mode",
+            "",
+            "Specifies which dynamic service discovery method to use. Currently we support only " +
+                    "\"zookeeper\" to specify ZooKeeper based service discovery."),
+    THRIFT_ZOOKEEPER_CLIENT_PORT("metastore.zookeeper.client.port",
+            "hive.metastore.zookeeper.client.port", "2181",
+            "The port of ZooKeeper servers to talk to.\n" +
+                    "If the list of Zookeeper servers specified in hive.metastore.thrift.uris" +
+                    " does not contain port numbers, this value is used."),
+    THRIFT_ZOOKEEPER_SESSION_TIMEOUT("metastore.zookeeper.session.timeout",
+            "hive.metastore.zookeeper.session.timeout", 120000L, TimeUnit.MILLISECONDS,
+            new TimeValidator(TimeUnit.MILLISECONDS),
+            "ZooKeeper client's session timeout (in milliseconds). The client is disconnected\n" +
+                    "if a heartbeat is not sent in the timeout."),
+    THRIFT_ZOOKEEPER_CONNECTION_TIMEOUT("hive.metastore.zookeeper.connection.timeout",
+            "hive.metastore.zookeeper.connection.timeout", 15L, TimeUnit.SECONDS,
+            new TimeValidator(TimeUnit.SECONDS),
+            "ZooKeeper client's connection timeout in seconds. " +
+                    "Connection timeout * hive.metastore.zookeeper.connection.max.retries\n" +
+                    "with exponential backoff is when curator client deems connection is lost to zookeeper."),
+    THRIFT_ZOOKEEPER_NAMESPACE("hive.metastore.zookeeper.namespace",
```

Comment:
"hive." should not be there to follow the same conventions as other configs.

Reply:
My bad. Changed per suggestion. Thanks for pointing it out.